tests/src/test/java/de/sovity/edc/e2e/ContractTerminationTest.java

```diff
@@ -291,6 +293,7 @@ List<DynamicTest> theDetailsAreMandatory(
                 })).toList();
     }
 
+    @DisabledOnGithub
```

Comment:
they don't work?

Reply:
They used to work until they started popping up in the logs with the usual unfixable timeout issue.
https://github.com/sovity/edc-ce/issues/870